### DISCUSSION TITLE
🧩 FEATURE REQUEST: Hide post button

### DISCUSSION THREAD

**Discussion by enkeli** [dailydotdev/daily#1011] [Feature Requests]
### Description

I would like to have a dismiss button for every post, similar to the "Hide this post" action inside the top right menu of every post, but more accessible for faster posts dismissal after reading the post title.

### Solution

Just adding a hide post button alongside the post menu button in the top right corner of each post item.

![Screen Shot 2021-04-14 at 7 32 40 PM](https://user-images.githubusercontent.com/624503/114805772-6d37fa00-9d58-11eb-8141-3e6d385cd134.png)

### Alternate Solutions

The button could be placed anywhere on the post item.

**Comment by idoshamun** [comment]
Hey @enkeli,

Thanks for the suggestion and for putting effort into creating this screenshot.
We plan to add configuration options for the card, which can definitely be one option.
I'll keep you posted on the progress. It might take a little while.

**Comment by imranpollob** [comment]
I must say, we need a "mark as read" button like Gmail. "Hide post" is under the "Report post" dropdown which feels somewhat negative. I don't wanna report/hide a post in most cases, I just want to mark the content as read because I feel like I already know the topic. Hope you get the point.

**Comment by idoshamun** [comment]
@pollmix thanks for the clarification. I get the point :)
I'll raise this issue in our next product meeting.

**Comment by imranpollob** [comment]
@idoshamun Thanks for your consideration.
